test/sde/sde_convergence_tests.jl

```diff
@@ -22,6 +22,8 @@ sim  = test_convergence(dts,prob,LambaEM(),trajectories=Int(1e2))
 @test abs(sim.𝒪est[:l2]-.5) < 0.2
```

Comment:
we need a non-commutative convergence test now. Does https://epubs.siam.org/doi/abs/10.1137/09076636X have one? IIRC it does

Reply:
Yes, It does. with m = 4. We should also find something similar for Stratonovich problems